src/main/java/io/antmedia/rest/StreamsSourceRestService.java

```diff
@@ -239,6 +273,15 @@ public Result updateCamInfo(Broadcast broadcast) {
 			}
 
 			result = getStore().editStreamSourceInfo(broadcast);
+			
```

Comment:
Functional or unit test code may be needed for this block as well?

Reply:
For which block?